modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -1866,10 +1866,11 @@ static inline double ratio(int inSz, int outSz) {
 }
 
 template<typename T, typename Mapper, int chanNum = 1>
-static inline void initScratchLinear(const cv::GMatDesc& in,
-                                     const         Size& outSz,
-                                     cv::gapi::fluid::Buffer& scratch,
-                                     int  lpi) {
+CV_ALWAYS_INLINE void initScratchLinear(const cv::GMatDesc& in,
```

Comment:
We don't usually use `CV_*` macros because it might broke `standalone` mode.
Consider of adding synonym to there: https://github.com/opencv/opencv/blob/4.x/modules/gapi/include/opencv2/gapi/own/exports.hpp

Reply:
Synonym was added to `own/assert.hpp`

Reply:
I believe the `assert.hpp` one is irrelevant here?

Reply:
Let's keep it as CV_ anyway.